docs/samples/Microsoft.ML.Samples/Dynamic/SsaChangePointDetectorTransform.cs

```diff
@@ -0,0 +1,89 @@
+    // the alignment of the usings with the methods is intentional so they can display on the same level in the docs site. 
+    using System;
+    using System.Linq;
+    using System.Collections.Generic;
+    using Microsoft.ML.Runtime.Data;
+    using Microsoft.ML.Runtime.Api;
+    using Microsoft.ML.Runtime.TimeSeriesProcessing;
```

Comment:
make indent consistent with other files, 8 spaces

Reply:
see my other comment. 

---
In reply to: [232375368](https://github.com/dotnet/machinelearning/pull/1578#discussion_r232375368) [](ancestors = 232375368)